pkgs/applications/editors/vscode/extensions/default.nix

```diff
@@ -602,8 +602,8 @@ let
           mktplcRef = {
             name = "catppuccin-vsc-icons";
             publisher = "catppuccin";
-            version = "0.12.0";
-            sha256 = "sha256-i47tY6DSVtV8Yf6AgZ6njqfhaUFGEpgbRcBF70l2Xe0=";
+            version = "0.30.0";
```

Comment:
```suggestion
            version = "1.10.0";
```

Reply:
Please, also add changelog. And description could just start with `A Visual Studio Code ...` to keep it consistent.

Reply:
Hi @superherointj, thanks for the review. I update the version and hash, though I left the description as is, to be consistent with `vscode-extensions.catppuccin.catppuccin-vsc` above.